capirca/lib/aclgenerator.py

```diff
@@ -137,6 +138,23 @@ def __init__(self, term):
                                            self.PROTO_MAP)
     self.term = term
 
+  def _IsValidProtocol(self, protocol):
+    """Return true if given protocol is a valid protocol in either
+    a known name format (string) or a valid numeric protocol.
+
+    Args:
+      protocol: Protocol to check. Normally should be either a str or int.
+
+    Returns:
+      Boolean True if protocol is valid, otherwise False
+    """
+    if protocol in self.PROTO_MAP:
+      return True
+    try:
+      return int(protocol) in self.ALL_PROTO_NUMBERS
```

Comment:
Since it's integre, maybe we don't need a set (defined in lines 116,117), but simply do the comparison here:
```
return 0 <= int(protocol) < 256
```